plugins/event_notification/providers/bpm/services/BusinessProcessCaseService.php

```diff
@@ -116,9 +116,7 @@ public function serveDiagramAction($objectType, $objectId, $businessProcessStart
 		$filename .= $businessProcessStartNotificationTemplateId . '_';
 		$filename .= $caseId . '.jpg';
 		
-		$url = $provider->getCaseDiagram($caseId, $filename);
-		
-		KCurlWrapper::getDataFromFile($url, $filename);
```

Comment:
Why did you remove this line? it is needed.

Reply:
Follow by discussion with Tan-Tan this can be remove . The second line getting an empty URL and therefor never set the file data